client/app/hoc/asUIBModal.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const asUIBModal = (WrappedComponent) => {
```

Comment:
Do you think this won't be used when we're 100% React? Because otherwise calling this `asUIBModal` is probably not the best name.

Also, are you sure this API is better over just `openModal()` function that will take the Modal to open along with whatever props we want it have?

Reply:
Actually, your way we only need to import the Modal...

Anyway, the important thing is the name :)

Reply:
@arikfr Indeed name is not super good, but we both with @ranbena  have no better idea ATM. But it be definitely used when we're 100% React.

> Also, are you sure this API is better over just openModal() function

It's more or less equal; this implementation is more OOP-way: `SomeModal.open(...)` vs. `openModal(SomeModal, ...)`

Reply:
> Do you think this won't be used when we're 100% React?

My intention is to remove the hoc once dashboard is reactified, but we'll discuss it then.

> Also, are you sure this API is better over just `openModal()` function that will take the Modal to open along with whatever props we want it have?

It's basically the same in every way. You either import the utility and the modal or the modal with the utility.

Reply:
I think we can merge it, but before that let's try to choose better name for this utility. How about `callerFriendlyModal`?